readme.md

```diff
@@ -204,7 +204,14 @@ Returns a TypeScript decorator which memoizes the given function.
 
 Type: `object`
 
-Same as options for `mem()`.
+Same as options for `mem()` and the following:
+
+##### isAcrossInstances
```

Comment:
When would this ever be useful? 🤷‍♂️ Instances are not meant to share a global states

Reply:
The only situation that we're not covering anywhere are static methods (isAcrossInstances doesn't apply here either)

Reply:
@fregante It's not something you would need often, but sometimes you do calculations that don't actually depend on any state in the instance. I don't have an immediate need for this though, just thought someone might find it useful. Do you think it's worth keeping it?

Reply:
> sometimes you do calculations that don't actually depend on any state in the instance

Isn't that the definition of a static method?

Reply:
Do decorators work on static methods? If so, we can indeed remove this option.

Reply:
The docs don't include any static examples, but it doesn't mention "static" a few times: https://www.typescriptlang.org/docs/handbook/decorators.html#decorator-evaluation

Also it seems to pass:

https://www.typescriptlang.org/play?#code/GYVwdgxgLglg9mABMGAnAzlAFASkQbwChFEIF04AbAUwDpK4BzLAIhQ2xwC5kBDaOKgCeiagDdelELyjUAJixwBuYolTUoIVElCRYCRFii9UjDT15ghAGkQAHVHDvVUUIQGlqQnplQwwjLZy1OgQfnZQgjwACo7OrkIAIiFhMBGCeEQkJGRgFDT0TKzsmLg8EJI0CsqqAL4qtYSEEJS86OiIAOLqGi4EqgACJZyqjD2c-dlqGlpILAAS1JQMLCokjSSDw7iqmDIwEIhj1Bq4k9nqmtqIC0sra4iNtUA

<img width="708" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/115375278-d8217f00-a1f7-11eb-989d-45d8211c939c.png">

Reply:
Alright let's remove the option and update the docs to mention support for static methods too, just to be user-friendly.

Reply:
@Richienb I just merged the other PR that extended the docs for this. Update this branch before extending the docs further.